api/v1alpha1/ippool_type.go

```diff
@@ -45,6 +45,10 @@ type IPPoolSpec struct {
 	Gateway string `json:"gateway,omitempty"`
 	// selector for nodes, if empty match all nodes
 	NodeSelector *corev1.NodeSelector `json:"nodeSelector,omitempty"`
+	// if true, add gateway as default gateway in the routes list
+	DefaultGateway *bool `json:"defautGateway,omitempty"`
```

Comment:
Do we really want to use a pointer here? It looks like that "false" is a safe default

Reply:
I agree with Yury, simple boolean should be enough I think.

Reply:
Done